dotCMS/src/main/java/com/dotcms/rest/api/v1/page/PageContainerForm.java

```diff
@@ -84,35 +84,69 @@ public PageContainerForm deserialize(final JsonParser jsonParser,
 
                 // Parse styleProperties for each contentlet (optional field)
                 final JsonNode stylePropertiesNode = jsonElement.get(STYLE_PROPERTIES_ATTRIBUTE_NAME);
-                if (stylePropertiesNode != null && stylePropertiesNode.isObject()) {
-                    stylePropertiesNode.fields().forEachRemaining(entry -> {
-                        final String contentletId = entry.getKey();
-                        final JsonNode styleProps = entry.getValue();
-                        if (styleProps != null && styleProps.isObject()) {
-                            final Map<String, Object> propsMap = new HashMap<>();
-                            styleProps.fields().forEachRemaining(prop -> {
-                                final JsonNode propValue = prop.getValue();
-                                // Handle different JSON value types
-                                if (propValue.isTextual()) {
-                                    propsMap.put(prop.getKey(), propValue.asText());
-                                } else if (propValue.isNumber()) {
-                                    propsMap.put(prop.getKey(), propValue.numberValue());
-                                } else if (propValue.isBoolean()) {
-                                    propsMap.put(prop.getKey(), propValue.asBoolean());
-                                } else {
-                                    propsMap.put(prop.getKey(), propValue.toString());
-                                }
-                            });
-                            containerEntry.setStyleProperties(contentletId, propsMap);
-                        }
-                    });
-                }
+                getStylePropertiesNode(stylePropertiesNode, containerEntry);
 
                 entries.add(containerEntry);
             }
 
             return new PageContainerForm(entries, jsonNode.toString());
         }
+
+        /**
+         * Gets the style properties node from the JSON object.
+         * @param stylePropertiesNode The JSON node containing the style properties.
+         * @param containerEntry The container entry to set the style properties.
+         */
+        private void getStylePropertiesNode(final JsonNode stylePropertiesNode, final ContainerEntry containerEntry) {
```

Comment:
This should probably get renamed to "setStylesProperties"
Something like that, I mean, something called "getSomerhing" is expected to return a value of some sort